src/midi/midi.cpp

```diff
@@ -252,18 +254,24 @@ class MIDI : public Module_base {
 		LOG_MSG("MIDI: Can't find device: %s, using default handler.",
 		        dev.c_str());
 getdefault:
-		handler=handler_list;
-		while (handler) {
+		for (handler = handler_list; handler; handler = handler->next) {
```

Comment:
This loop refactoring is very easy to read, and I like how the loop management is together. Very minor but a nice touch!